experiments/react-native-uifabric/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View, Text } from 'react-native';
+import {
+  radioGroupName,
+  IRadioGroupType,
+  IRadioGroupProps,
+  IRadioGroupState,
+  IRadioGroupSlotProps,
+  IRadioGroupRenderData,
+  IRadioGroupContext
+} from './RadioGroup.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { filterViewProps } from '../../utilities/RenderHelpers';
+import { settings } from './RadioGroup.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { foregroundColorTokens, textTokens } from '../../tokens';
+import { useAsRadioGroupSelection } from '../../hooks';
+
+export const RadioGroupContext = React.createContext<IRadioGroupContext>({
+  selectedKey: '',
+  onButtonSelect: (key: string) => {
+    return;
+  }
+});
+
+export const RadioGroup = compose<IRadioGroupType>({
+  displayName: radioGroupName,
+
+  usePrepareProps: (userProps: IRadioGroupProps, useStyling: IUseComposeStyling<IRadioGroupType>) => {
+    const { label, ariaLabel, ...rest } = userProps;
+
+    // This hook updates the Selected Button and calls the customer's onClick function. This gets called after a button is pressed.
+    const data = useAsRadioGroupSelection(userProps.defaultSelectedKey ? userProps.defaultSelectedKey : '', userProps.onChange);
+
+    const state: IRadioGroupState = {
+      selectedKey: data.selectedKey,
```

Comment:
Maybe it's not a big deal, but I'd consider putting these inside an 'info' or 'context' subobject like Button does, or changing the override lookup that useStyling is doing.  As it is, if you added a "selectedKey" or "onChange" override in your settings those values would always get applied.  Unlikely to happen, but a little weird.

Reply:
Fixed in my newest commit